### PR TITLE
log-backup: update version check for pitr

### PR DESCRIPTION
**Pull request by Leavrth**
Signed-off-by: Leavrth <jianjun.liao@outlook.com>

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.3 (TiDB 6.3 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by IANTHEREAL** [comment]
LGTM

File: br/point-in-time-recovery.md [hunk 1]
```diff
@@ -92,6 +92,14 @@ PITR 可用于满足以下业务需求：
 - 不支持在恢复中重复恢复某段时间区间的日志，如果多次重复恢复 [t1=10, t2=20) 区间的日志备份数据，可能会造成恢复后的数据不一致。
 - 其他已知问题，请参考 [PITR 已知问题](/br/pitr-known-issues.md)。
 
+### 版本兼容检查
+
+v6.3 对 PITR 产生的备份文件做了一些改动，例如新的压缩方法和小文件的合并，使得无法向前兼容。
+| 恢复版本（横向）\ 备份版本（纵向）   | 用 PiTR v6.2 恢复 TiDB v6.2 | 用 PiTR v6.3 恢复 TiDB v6.3 |
+|  ----  |  ----  | ---- |
+|用 PiTR v6.2 备份 TiDB v6.2| 兼容 | 兼容 |
+|用 PiTR v6.3 备份 TiDB v6.3| 不兼容 |兼容 |
```
Comment (shichun-0415) [inline br/point-in-time-recovery.md:101]: 一些改动还是一个改动？改动是不是就只有新的压缩方法和小文件的合并？如果是，建议：
> 在 v6.3.0，PITR 产生的备份文件采用新的压缩方法，同时还会合并小文件，解决了之前小文件过多带来的问题，但同时也导致无法向前兼容。
```suggestion
v6.3.0 对 PITR 产生的备份文件做了一些改动，例如新的压缩方法和小文件的合并，使得无法向前兼容。
| 恢复版本（横向）\ 备份版本（纵向）   | 用 PITR v6.2.0 恢复 TiDB v6.2.0  | 用 PITR v6.3.0 恢复 TiDB v6.3.0  |
|  ----  |  ----  | ---- |
|用 PITR v6.2.0 备份 TiDB v6.2.0 | 兼容 | 兼容 |
|用 PITR v6.3.0 备份 TiDB v6.3.0 | 不兼容 |兼容 |
```

**Review by joccau — APPROVED** [review]
LGTM

**Review by shichun-0415 — APPROVED** [review]
LGTM

File: br/point-in-time-recovery.md [hunk 2]
```diff
@@ -92,6 +92,14 @@ PITR 可用于满足以下业务需求：
 - 不支持在恢复中重复恢复某段时间区间的日志，如果多次重复恢复 [t1=10, t2=20) 区间的日志备份数据，可能会造成恢复后的数据不一致。
 - 其他已知问题，请参考 [PITR 已知问题](/br/pitr-known-issues.md)。
 
+### 版本兼容检查
+
+在 v6.3.0，PITR 产生的备份文件采用新的压缩方法，同时还会合并小文件，解决了之前小文件过多带来的问题，但同时也导致无法向前兼容。
```
Comment (TomShawn) [inline br/point-in-time-recovery.md:97]:
```suggestion
在 v6.3.0 中，PITR 产生的备份文件采用新的压缩方法，同时还会合并小文件（为解决之前小文件过多带来的问题），这会导致向前兼容性问题。

```

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**